confluent-kafka-plugins/widgets/Confluent-streamingsource.json

```diff
@@ -174,8 +175,19 @@
   "outputs": [
     {
       "name": "schema",
+      "label": "schema",
       "widget-type": "schema",
       "widget-attributes": {
+        "schema-types": [
```

Comment:
Any particular reason to set these schema-types explicitly ? Any type which is not supported?

Reply:
I don't recall any reason as such but this is the process that we follow in every plugin. 

All the given types are supported. [Avro Primitive Types](https://avro.apache.org/docs/1.11.1/specification/_print/)

Reply:
I checked on the use of `schema-types` for widget JSON. This was intended to restrict schema types for plugins. This however does not work in UI at the moment - https://cdap.atlassian.net/browse/CDAP-20534 . Considering that previous versions of the plugin did not restrict this, we should not introduce this here. If some types are unsupported, let's create a new JIRA and handle it separately in plugin validation.

Reply:
Removed schema types. fc7272c4bb7df42e57e27a6f16230f9aeecf0263